tests/unit/oauth_test_utils.py

```diff
@@ -45,9 +45,15 @@ def __call__(self, request, uri, response_headers):
         authorization = request.headers.get("Authorization")
         if authorization and authorization.replace("Bearer ", "") in self.tokens:
             return [200, response_headers, json.dumps(self.sample_post_response_data)]
-        return [401, {'Www-Authenticate': f'Bearer x_redirect_server="{self.redirect_server}", '
-                                          f'x_token_server="{self.token_server}"',
-                      'Basic realm': '"Trino"'}, ""]
+        return [
+            401,
+            {
+                "Www-Authenticate": f'Bearer x_redirect_server="{self.redirect_server}", '
+                f'x_token_server="{self.token_server}"',
+                "Basic realm": '"Trino"',
```

Comment:
More readable and "safer" to have entire token on one line

Reply:
Fixed by increasing line length.